equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/mapper/Crm14Mapper.java

```diff
@@ -207,6 +205,12 @@ public interface Crm14Mapper extends CrmMapper {
     @Mapping(target="partnerEmployed", source = "partner_employed")
     Crm14EvidencePart1DTO getEvidencePart1DTOFromModel(Crm14DetailsModel model);
 
+    @Mapping(target="offenceDate1", source="offence_date_1")
+    @Mapping(target="offenceDate2", source="offence_date_2")
+    @Mapping(target="offenceDate3", source="offence_date_3")
+    @Mapping(target="charge", source="charge")
+    @Mapping(target="whenOffence", source="offence_when")
```

Comment:
These names are misleading.  Any better alternative than 1, 2, 3 and when?  What's the difference?

Reply:
modified with appropriate names